johnsonandjohnson/Bodiless-JS
# fix (backend): Completely delete new pages on revert

## Changes
- Make sure new pages are completely deleted after resetting changes.
- Clean up deleted pages from the 'public' directory.

## Test Instructions
- create some new pages (with data or not)
- revert changes using UI button
- make sure the pages are deleted, page paths return 404, new pages can be created on the paths

## Related Issues
Fixes #125
